### ISSUE TITLE
Hydration double data: import '*.css'

### ISSUE THREAD

**Issue by btakita** [solidjs/solid-start#687]
Importing css results in a `<style>` tag in ssr & in hydration. Since the `<style>` tag is already rendered in ssr, is hydrating the css file necessary?

```tsx
import 'somefile.css'
```

**Comment by btakita** [comment]
Using the `@tailwind` directive in the project's `index.css` also causes hydration double data.

```css
@tailwind base;
@tailwind components;
@tailwind utilities;
```

**Comment by ryansolid** [comment]
Yeah this only happens in dev right? This is a result of us trying to fix the really bad FOUC that happens in dev mode. Tailwind processing time has other issues too but I think this may be a duplicate.

**Comment by nksaraf** [comment]
We can try to avoid hydration, but I think we really need to do this setup during dev coz the FOUC is much worse experience than the styles appearing twice.

**Comment by btakita** [comment]
It looks like the double hydration is not happening in production. It's ok once the developer is aware of the behavior. Is this behaviour a well defined part of solid-start, as in it should have docs calling it out?

**Comment by nksaraf** [comment]
Yeah probably thats a good point. It wont happen in production ever. Only a dev thing and shouldnt create any side effects except if the developer is relying on some weirdly css declaration ordering behaviour (which they shouldn't) lol. We should add it to the docs calling it out probably
